.github/settings.yml

```diff
@@ -183,7 +183,7 @@ branches:
         # Required. Require branches to be up to date before merging.
         strict: true
         # Required. The list of status checks to require in order to merge into this branch
-        contexts: ["test (1.x, ubuntu-latest) / ./ubuntu-latest/1.x"]
+        contexts: ["test (1.24.x) / test: /1.24.x"]
```

Comment:
This is correct, and we are going to change it soon because we refactored the pipeline so this check is outdated. See https://github.com/testcontainers/testcontainers-go/issues/2968